boxsdk/object/api_json_object.py

```diff
@@ -0,0 +1,27 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+from collections import Mapping
+
+# import six
+
+from .base_api_json_object import BaseAPIJSONObject, BaseAPIJSONObjectMeta
+
+
+class APIJSONObjectMeta(BaseAPIJSONObjectMeta, Mapping.__metaclass__):
```

Comment:
Rather than using `Mapping.__metaclass__` (which won't work on Python 3), I would instead recommend either `Mapping.__class__`, `type(Mapping)`, or, since we already know what its metaclass is, `ABCMeta`.